src/josepy/jws.py

```diff
@@ -361,13 +373,13 @@ def sign(cls, args):
             print(sig.json_dumps_pretty())
 
     @classmethod
-    def verify(cls, args):
+    def verify(cls, args: argparse.Namespace) -> int:
```

Comment:
I think this should probably be:
```suggestion
    def verify(cls, args: argparse.Namespace) -> bool:
```
and the `return -1` below is converted to `return False`. What do you think?

Reply:
Indeed!